manual/parts/intro.tex

```diff
@@ -15,7 +15,7 @@ \chapter{\label{ch:intro}Introduction}
 spaces, threads, and inter-process communication (IPC). The small number
 of services provided by seL4 directly translates to a small
 implementation of approximately $8700$ lines of C code. This has allowed
-the ARMv6 version of the kernel to be formally proven in the Isabelle/HOL 
+the ARMv6 version of the kernel to be formally proved in the Isabelle/HOL
```

Comment:
nitpicking: change does not fit to commit message ;)

Reply:
;-)